src/pages/FirstRun.vue

```diff
@@ -88,7 +89,13 @@ export default Vue.extend({
     defaultVersion(): VersionEntry {
       const version = this.recommendedVersions.find(v => (v.channels ?? []).includes('stable'));
 
-      return version ?? (this.recommendedVersions ?? this.nonRecommendedVersions)[0];
+      return version ?? this.recommendedVersions[0] ?? this.nonRecommendedVersions[0];
+    },
+    // This field is needed because the template-parser doesn't like `defaultVersion?.version.version`
+    unwrappedDefaultVersion(): string {
+      const wrappedSemver = this.defaultVersion;
+
+      return wrappedSemver ? wrappedSemver.version.version : '';
```

Comment:
Yes, but I found all the variants of `version` in your alternative less informative. The key is that `this.defaultVersion` is either undefined or a wrapped semver. If it's the latter, unwrap it once with the first `.version` field, and then get the "cooked" `version` string with the second `version` field. Otherwise just return the empty string.